public_html/js/mpform.js

```diff
@@ -0,0 +1,79 @@
+//jQuery time
+var current_fs, next_fs, previous_fs; //fieldsets
+var left, opacity, scale; //fieldset properties which we will animate
+var animating; //flag to prevent quick multi-click glitches
+
```

Comment:
These variables pollute the global namespace. If they manage state, then stick them in the `PIN.MPF` namespace (or similar) to keep them from colliding elsewhere. Remember this is a library that can be used on any web page anywhere, so respect namespaces.